clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -674,8 +674,10 @@ static Attr *ProcessStmtAttribute(Sema &S, Stmt *St, const ParsedAttr &A,
          A.existsInTarget(*Aux)))) {
     if (A.isRegularKeywordAttribute() || A.isDeclspecAttribute()) {
```

Comment:
I think I agree with corentin, just breaking this up into 2 branches (an `if (A.isRegularKeywordAttribute())` and `else if (A.isDeclspecAttribute())` is probably better.  As it is, we're basically checking the regular keyword 2x.

Else, this patch is fine for me.